src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no,  initial-scale=1.0, maximum-scale=1.0, minimum-scale=1. 0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+  </head>
+  <body>
+    <div class="card card--main" data-qa="card">
+      <div class="card__pic_container">
+        <img
+          class="card__pic" 
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K   
+          Monoblock"
+        >
+      </div>
+
+      <div class="card__about card__about--font">
+        APPLE A1419 iMac 27" Retina 5K   
+        Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__prod_id card__prod_id--font">
+        Product code: 195434
+      </div>
+
+      <div class="stars">
+        <ul class="stars__average stars__average--4">
+          <li class="stars__star"></li>
+
```

Comment:
don't bloat the code with extra spaces. They should be left when the blocks are separated
```suggestion
```

Reply:
ok, i will fix it... but i did it  according to the checklist 
![image](https://user-images.githubusercontent.com/70288388/201389022-7675283d-70bb-4712-8278-aab5fe15271a.png)